src/App.js

```diff
@@ -4,13 +4,17 @@ import { Routes, Route } from 'react-router-dom';
 
 import Navbar from './components/Navbar/index';
 import Homepage from './Pages/Homepage';
+import Product from './Pages/Products';
+import Wishlist from './Pages/Wishlist';
```

Comment:
suggestion: add index.js file in pages and import and export all pages from the index.js itself, so the number of code lines will decrease.